felix/bpf/nat/maps.go

```diff
@@ -249,18 +249,20 @@ func (v FrontendValue) Flags() uint32 {
 
 func (v FrontendValue) FlagsAsString() string {
 	flgs := v.Flags()
+	if flgs == 0 {
+		return ""
+	}
+
 	fstr := ""
 
-	for i := 0; i < 32; i++ {
-		flg := uint32(1 << i)
-		if flgs&flg != 0 {
-			fstr += flgTostr[int(flg)]
-		}
-		flgs &= ^flg
-		if flgs == 0 {
-			break
+	for flg, str := range flgTostr {
```

Comment:
Map iteration is non-deterministic so this will come out in random order.  I think it's pretty important to have deterministic string functions.  Makes it a lot easier to scan/diff/test.